flexmeasures/data/models/planning/__init__.py

```diff
@@ -34,26 +36,38 @@ class Scheduler:
     __version__ = None
     __author__ = None
 
-    sensor: Sensor
+    sensor: Optional[Sensor] = None
+    asset: Optional[Asset] = None
+
+    asset_or_sensor: Asset | Sensor
```

Comment:
This is an `__init__` argument, but not actually stored (nor used) as a class property, so it can be removed here.

Reply:
Good that you noticed. Setting the value or sensor or asset_or_sensor

Reply:
Are we using the class property anywhere? Else, why do we need it?

Reply:
Not used by now. I though of adding it to have it available from the `.compute()` method, specially, anticipating for the `MultiDeviceStorageScheduler`. I think we can remove it for now and introduce it in case we need it on the future.